web/src/main/webapp/WEB-INF/classes/setup/sql/migrate/v443/migrate-default.sql

```diff
@@ -4,3 +4,13 @@ INSERT INTO Settings (name, value, datatype, position, internal) VALUES ('system
 
 UPDATE Settings SET value='4.4.3' WHERE name='system/platform/version';
 UPDATE Settings SET value='0' WHERE name='system/platform/subVersion';
+
+CREATE TABLE spg_page_group
```

Comment:
@wangf1122 
I'm not sure if create tables are required in the migration as I think JPA will take care of creating any missing tables.

@josegar74
Can you confirm if this is required?

Reply:
@josegar74 @ianwallen 

I had confirmed myself. It is not required. The JPA will automatically create the table. I will update the script.